python/src/MaximumLikelihoodFactory_doc.i.in

```diff
@@ -148,3 +148,45 @@ Returns
 -------
 indices : :class:`~openturns.Indices`
     Indices of fixed parameters."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::MaximumLikelihoodFactory::BuildGaussianEstimator
+"Compute distribution of parameters.
+
+Compute the asymptotic distribution of parameters for a regular distribution via the delta method.
+
+Parameters
+----------
+distribution : :class:`~openturns.Distribution`
+    Inferred distribution
+sample : 2-d sequence of float
+    Data to infer
+
+Returns
+-------
+distribution : :class:`~openturns.Distribution`
+    Asymptotic gaussian distribution of the parameters."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::MaximumLikelihoodFactory::BuildEstimator
+"Compute distribution of parameters.
+
+Compute the asymptotic distribution of parameters.
+
+Parameters
+----------
+factory : :class:`~openturns.DistributionFactory`
+    Distribution factory to infer the data
+sample : 2-d sequence of float
+    Data to infer
+is_regular : bool
+    Whether the distribution is known to be regular
```

Comment:
```suggestion
    Whether the distribution is known to be regular.
```

Reply:
I agree that the doc has to be improved even if the requested changes do not affect the validity of the content. This can be done during the RC phase. Splitting this PR into 2 PR generates much work. It does not appear necessary to me.

Reply:
ok